docs/monorepo.md

```diff
@@ -128,8 +137,8 @@ The Jetpack Monorepo includes GitHub actions to run a number of CI checks on all
 
 Tests for a project are only run for a PR if changes are made to the project or its dependencies. Dependencies may be specified as:
 
-* For Composer packages included in the monorepo, via `.require' and `.require-dev` in `composer.json`.
-* For any other dependencies, via `.extra.dependencies` in `composer.json`.
+- For Composer packages included in the monorepo, via `.require' and`.require-dev`in`composer.json`.
```

Comment:
This line was broken already (thanks to the single-quote rather than backquote after `.require`), but you broke it more by removing spaces.

Reply:
🤦 It's an VScode's automated Markdown linting. I'll revert these changes